README.md

```diff
@@ -51,6 +51,13 @@ Optional flags:
 ```
 See https://hub.docker.com/repository/docker/axelarnet/axelar-core and https://hub.docker.com/repository/docker/axelarnet/tofnd for the latest available versions of the docker images.
 
+You can get the latest version and save it to variables:
```

Comment:
i'm not sure we should always fetch the latest version. we might push a release internally, which is first tested on a devnet, and then goes into a testnet. perhaps the release #s should be fetched from the TESTNET RELEASE file in the repo.

Reply:
@sergeynog ok updated the code to use TESTNET RELEASE file now


$ TOFND_VERSION=$(curl -s https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/TESTNET%20RELEASE.md | grep tofnd | cut -d \` -f 4)
$ CORE_VERSION=$(curl -s https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/TESTNET%20RELEASE.md | grep axelar-core | cut -d \` -f 4)
$ echo ${TOFND_VERSION} ${CORE_VERSION}
v0.2.0 v0.3.3